### DISCUSSION TITLE
Bad error message for email during verification

### DISCUSSION THREAD

**Discussion by michnovka** [danny-avila/LibreChat#3904] [Troubleshooting]
### What happened?

When I try to sign up user, I get an error There was an error attempting to register your account. Please try again. Something went wrong

### Steps to Reproduce

1. enable ALLOW_REGISTRATION=true in .env
2. go to /register URL
3. enter all fields
4. click on Continue

this is my .env for registration values:
```

ALLOW_EMAIL_LOGIN=true
ALLOW_REGISTRATION=true
ALLOW_SOCIAL_LOGIN=false
ALLOW_SOCIAL_REGISTRATION=false
ALLOW_PASSWORD_RESET=false
# ALLOW_ACCOUNT_DELETION=true # note: enabled by default if omitted/commented out
ALLOW_UNVERIFIED_EMAIL_LOGIN=true

```

### Relevant log output

```shell
LibreChat         | 2024-09-02 19:27:40 error: [registerUser] Error in registering user: self-signed certificate in certificate chain
LibreChat         | 2024-09-02 19:27:40 warn: [registerUser] [Email: XXXX@XXX.XXX] [Temporary User deleted: {"deletedCount":1,"message":"User was deleted successfully."}]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by michnovka** [comment]
After some investigation, I found it was self signed cert in email:

```

EMAIL_HOST=10.0.0.1
EMAIL_PORT=25
EMAIL_ENCRYPTION=
EMAIL_ENCRYPTION_HOSTNAME=
EMAIL_ALLOW_SELFSIGNED=1
```

and should have been

```
EMAIL_ALLOW_SELFSIGNED=true
```

Thus the cert was not recognized. The error message should make it clear it is about e-mail, as it was confusing and first I thought it has to do with website https cert

**Reply by danny-avila** [reply]
The error is generated by [nodemailer](https://nodemailer.com/), PRs welcome for handling it.
